.github/workflows/main.yml

```diff
@@ -9,18 +9,18 @@ jobs:
   test:
     strategy:
       matrix:
-        os: ['ubuntu-18.04', 'macOS-10.15', 'windows-2019']
+        os: ['ubuntu-18.04', 'macos-10.15', 'windows-2019']
```

Comment:
`macos` is correct: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on